authnz/src/main/java/io/stargate/auth/AuthorizationService.java

```diff
@@ -32,6 +33,7 @@
    * @param typedKeyValues A list of {@link TypedKeyValue} that will be used in the query and should
    *     be authorized against the token.
    * @param sourceAPI The source api which calls this method.
+   * @param headers The map of headers propagated from the request (HTTP, GraphQl, ...)
```

Comment:
Should probably clarify that this pertains to headers for the HTTP services and properties for CQL. This comment implies these are only necessary for rest/graphql.

Reply:
done